internal/services/containers/client/client_gen.go

```diff
@@ -3,25 +3,16 @@ package client
 import (
 	"fmt"
 
-	containerserviceV20230302Preview "github.com/hashicorp/go-azure-sdk/resource-manager/containerservice/2023-03-02-preview"
 	containerservice_2024_04_01 "github.com/hashicorp/go-azure-sdk/resource-manager/containerservice/2024-04-01"
 	"github.com/hashicorp/go-azure-sdk/sdk/client/resourcemanager"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/common"
 )
 
 type AutoClient struct {
-	V20230302Preview containerserviceV20230302Preview.Client
-	V20231015        containerservice_2024_04_01.Client
+	V20231015 containerservice_2024_04_01.Client
```

Comment:
```suggestion
	V20240401 containerservice_2024_04_01.Client
```

Please update the version in the clients below as well

Reply:
Got it! Updated.

Reply:
Hi @stephybun , I just reverted my changes, because it will affect the generated code: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/containers/kubernetes_fleet_member_resource_gen.go#L77

Reply:
Thanks for the info. I've updated this PR.